src/components/chat-item/chat-item-tree-file.ts

```diff
@@ -143,8 +143,10 @@ export class ChatItemTreeFile {
   };
 
   public readonly hideTooltip = (): void => {
-    clearTimeout(this.fileTooltipTimeout);
-    if (this.fileTooltip !== null) {
+    if (this.fileTooltipTimeout != null) {
+      clearTimeout(this.fileTooltipTimeout);
+    }
+    if (this.fileTooltip != null) {
```

Comment:
Not blocking, but I still don't think these `if` statements have any effect?

Reply:
Hmm, hideTooltip can be called because of the mouse leave/out event fired while user passing their mouse over the screen. Since because of the tooltip delay, when the hideTooltip fired the tooltip component might not be there yet, which may cause an error to throw.

Reply:
Done